webapp/loggedin/modules/data/dataVis/dataQuery/components/tableContent/tableColumn.js

```diff
@@ -32,6 +32,15 @@ const TableColumn = (props) => {
     return `surveyForm.${nodeDefTypePrefix}.${colName}`
   }
 
+  const getColValue = (col) => {
+    const value = Object.prototype.hasOwnProperty.call(row, col) ? row[col] : null
+    if (value && NodeDef.isDecimal(nodeDef)) {
+      // Round decimal values to 2 decimal digits
+      return Number(value).toFixed(2)
```

Comment:
should have a ```NumberUtils.round = (number, digits = 2)``` function ?

Reply:
ok, but maybe we can use a default value of 0 for the decimal digits because usually a round function "returns the value of a number rounded to the nearest integer". Also, in that case there can be some inconsistencies between browser when using Number.toFixed (see [this](https://stackoverflow.com/questions/566564/math-roundnum-vs-num-tofixed0-and-browser-inconsistencies))

Reply:
not sure why it should default to 0.
at this point I think it's better to format both, decimal and integer.
Let's add bignumber.js library and use space for group separators and comma for decimal. We'll set up internationalization later.
see https://mikemcl.github.io/bignumber.js/#toFor

Reply:
and we have NuberUtils.formatDecimal and formatInteger.

here the config
```
BigNumber.config({
  ERRORS: false,
  FORMAT: {
    decimalSeparator: '.',
    groupSeparator: ' ',
    groupSize: 3
  }
})
```

Reply:
ok good